### PR TITLE
Enable ShellCheck linting, fix uncovered warnings in scripts

### PR DESCRIPTION
**Pull request by gicmo**
Inspired by osbuild/osbuild-composer#989

### CODE REVIEW

File: run-tests.sh [hunk 1]
```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/bash
 set -euo pipefail
 
+SHELLCHECK_SEVERITY=${SHELLCHECK_SEVERITY:-warning}
 
 run_test() {
-  podman run -it -v "$(pwd)":/github/workspace:z --env "GITHUB_WORKSPACE=/github/workspace" koji.test "$1"
+  podman run --rm -it -v "$(pwd)":/github/workspace:z --env "GITHUB_WORKSPACE=/github/workspace" koji.test "$1"
```
Comment (larskarlitski) [inline run-tests.sh:7]: Did you want to ad this `--rm`? Seems unrelated to this commit.

**Review by larskarlitski — APPROVED** [review]
Nice tool! Thanks :)

File: run-tests.sh (hunk 1 above)
Comment (gicmo) [reply run-tests.sh:7]: :shushing_face:  :upside_down_face: Should be fixed ;)

**Review by larskarlitski — APPROVED** [review]
Lovely, thanks!
